perftest/simulations/README.md

```diff
@@ -27,7 +27,7 @@ class SimpleSimulation extends Simulation {
     .client(
       NessieClient
         .builder()
-        .withUri("http://127.0.0.1:19120/api/v1")
+        .withUri(s"${System.getProperties.getProperty("nessie.uri")}")
```

Comment:
Not needed (same for the ones below)

Reply:
Correct - not neeed - the client builder automatically uses the value of the `nessie.uri` system property.

Reply:
nice! I didn't know that. Changed.